chat-api/v2/README.md

```diff
@@ -1,5 +1,34 @@
 # Release notes
 
+## 2.11
+
+This release introduces bulk messaging on the Chat API.  
+
+### Features
+
+#### Bulk messaging
+
+Beginning with this release the Chat API supports bulk messaging.
+
+Bulk messaging enables the business to send multiple messages in order to one user without the need of waiting for
+delivery receipts for single messages.
+
+Between 2 and 20 messages on all supported channels - SMS, WhatsApp and Viber - can be chained.
+
+All content types that a specific channel supports are supported as well.
+
+**Note** On WhatsApp the 24h support window rule is still effective when using this feature.
+
+#### WhatsApp : Location with URL
+
+When a user sends a location to the business, the URL of the origin site is present as well.
+
+#### WhatsApp : WhatsApp business account id (WABA ID)
+
+The account model of WhatsApp transports the WhatsApp business account id (WABA ID) as well.
+
+This id is generated by WhatsApp, when an account is created. 
+
```

Comment:
Tiny improvements suggestions:

#### Bulk messaging

Beginning with this release, the Chat API supports bulk messaging.

Bulk messaging enables a business to send multiple messages in order to a single user without the need to wait for
delivery receipts for individual messages.

Between 2 and 20 messages can be chained on all supported channels - SMS, WhatsApp, and Viber.

All content types that a specific channel supports are supported.

**Note** On WhatsApp the 24-hour support window policy is still effective when using this feature.

#### WhatsApp - Location with a URL

When a user sends a location to the business, the URL of the origin site is present as well.

#### WhatsApp - WhatsApp business account id (WABA ID)